### ISSUE TITLE
Empty keys inside getAllCountsByColumn sql query

### ISSUE THREAD

**Issue by DylanPetrey** [benvinegar/counterscale#126]
I haven't opened this project in a while. After coming back, I selected the yesterday filter and the page crashed. Looking at the network tab, I noticed the following request with a 422 response.

#### URL
`https://api.cloudflare.com/client/v4/accounts/[CF_ACCOUNT_ID]/analytics_engine/sql`

#### Response
```text
Input was invalid: sql parser error: Expected an expression:, found: )
```

I logged the query for this page and this is what I saw 
```sql
SELECT blob3,
    double1 as isVisitor,
    SUM(_sample_interval) as count
FROM metricsDataset
WHERE timestamp >= toDateTime('2024-12-12 06:00:00') AND timestamp < toDateTime('2024-12-13 06:00:00')
    AND blob3 IN ()
    AND double1 = 0
    AND blob8 = 'counterscale-dev'
GROUP BY blob3, double1
ORDER BY count DESC
LIMIT 10
```

There is an empty array inside the where statement
```sql
AND blob3 IN ()
```

This happens when there are no keys in the array

**Comment by DylanPetrey** [comment]
I already have a fix for this, just wanted to create an issue beforehand

**Comment by benvinegar** [comment]
Ah, nice catch.

**Comment by benvinegar** [comment]
Closed via #127.
